app/views/LicensorViewComponent.vue

```diff
@@ -146,16 +151,20 @@ div.licensor.container(v-else)
       th.border ID
       th.border Slug
       th.border Name
+      th.border License days granted to client
       th.border minimumLicenseDays
-      th.border No of license days assigned by client
-      th.border No of users to whom licenses have been assigned
+      th.border License days used by client
+      th.border License days remaining
+      th.border No of users having active licenses
```

Comment:
Probably less confusing to remove `No of `, and just say `Users having active licenses`